README.md

```diff
@@ -2,86 +2,138 @@
+  const taskFn = async () => { /** Do something... */ };
 
-otherPoller({
-  taskFn,
-  interval: 500,
-  timeout: 5000
-  // Other options... 
-})
+  otherPoller({
+    taskFn,
+    interval: 500,
+    timeout: 5000
+    // Other options... 
+  });
 ```
 
 you can achieve the same effect with this syntax: 
 
 ```
-import { poll } from 'pollsky';
+  import { poll } from 'pollsky';
 
-const taskFn = async () => { /** Does something and returns a string */ };
+  const taskFn = async () => { /** Does something and returns a string */ };
 
-const conditionFn = value => value === 'foo';
+  const conditionFn = value => value === 'foo';
 
-const poll(taskFn).atMost(5000, 'milliseconds').withInterval(500, 'milliseconds').until(conditionFn);
+  const poll(taskFn).atMost(5000, 'milliseconds').withInterval(500, 'milliseconds').until(conditionFn);
 ```
 
 ## Installation
 
 Using npm:
 
 ```
-$ npm install pollsky
+  $ npm install pollsky
 ```
 
 Using yarn:
 
 ```
-$ yarn add pollsky
+  $ yarn add pollsky
 ```
 
 ## Usage
 
 The simpliest use case:
 
 ```
-poll(waitForSomething).until(checkCondition);
+  poll(waitForSomething).until(checkCondition);
 ```
 
 where `waitForSomething` is an async function to keep executing and `checkCondition` - a function that checks if polling has ended successfully.  
 
 By default *Pollsky* does not call timeout and is being executed without the end. If you want to change this behaviour you can define a timeout like this:
 
 ```
-// In seconds...
-poll(waitForSomething).atMost(20, 'seconds').until(checkCondition);
+  // In seconds...
+  poll(waitForSomething).atMost(20, 'seconds').until(checkCondition);
 
-// ...and in milliseconds if you like
-poll(waitForSomething).atMost(500, 'milliseconds').until(checkCondition);
+  // ...and in milliseconds if you like
+  poll(waitForSomething).atMost(500, 'milliseconds').until(checkCondition);
 ```
 
 Using `withInterval()` we can change the polling interval:
 ```
-poll(waitForSomething).withInterval(5, 'seconds').until(checkCondition);
+  poll(waitForSomething).withInterval(5, 'seconds').until(checkCondition);
 
-// We can easily chain methods however we want
-poll(waitForSomething).withInterval(5, 'seconds').atMost(2, 'minutes').until(checkCondition);
+  // We can easily chain methods however we want
+  poll(waitForSomething).withInterval(5, 'seconds').atMost(2, 'minutes').until(checkCondition);
 ```
 
 It's sometimes useful to ignore exceptions during condition evaluation.
 ```
-poll(waitForSomething).ignoreErrors().until(checkCondition);
+  poll(waitForSomething).ignoreErrors().until(checkCondition);
 ```
 
 You can instruct *Pollsky* to wait a certain amount of time
 ```
-poll(waitForSomething).atMost(30, 'seconds').until(checkCondition);
+  poll(waitForSomething).atMost(30, 'seconds').until(checkCondition);
+```
+
+If we don't want to *Pollsky* to throw an error at the very end if polling failed we can use `dontThrowError()` to return the last result
```

Comment:
Unnecessary 'to' before 'Pollsky'
Instead of 'at the very end if polling failed' -> 'when polling fails'